modules/apps/object/object-test/src/testFunctional/tests/ObjectExportImport.testcase

```diff
@@ -622,6 +622,89 @@ definition {
 			value1 = "Approved");
 	}
 
+	@description = "LPS-194547 - Verify it is possible to update a definition using the import"
+	@priority = 5
+	test CanUpdateDefinitionWithImport {
+		property custom.properties = "feature.flag.LPS-148856=true";
+		property portal.acceptance = "true";
```

Comment:
Hey @JhossephG, let's add this to acceptance after we see that this test is stable on CI. Let's remove this property for now.